### PR TITLE
Added a new section Backup and restore of Redis

### PR DESCRIPTION
**Pull request by sunithasomanna**
Added a new section Backup and restore of Redis in the Getting Started Readme

### CODE REVIEW

**Review by amar-shalgar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2326,11 +2327,166 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
 
     
 
+## Backup and restore of Redis
 
+1. Create a directory to store the backup rdb files and append-only files.
 
+   ```
+   mkdir backup
```
Comment (ashraf-vazeer) [inline README.md:2335]: update as follows

mkdir <backup dir>/backup

File: README.md [hunk 2]
```diff
@@ -2326,11 +2327,166 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
...
+   ```
+   ```
+   redis-ha-ondisk-primary-0
+   ```
+
+3. Copy the rdb files and aof files from the master pod.
+
+   ```
+   kubectl cp odim/<pod-name>:/redis-data/dump.rdb /home/bruce/backups/dump.rdb
+   ```
```
Comment (ashraf-vazeer) [inline README.md:2357]: where ever there is /home/bruce is present, replace  with the following

<backup dir>/

for e.g. /home/bruce/backups/dump.rdb  

change to

<backup dir>/backups/dump.rdb

File: README.md [hunk 3]
```diff
@@ -2326,11 +2327,166 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
...
+kubectl cp odim/redis-ha-inmemory-primary-0:/redis-data/appendonly.aof /home/bruce/backups/appendonly.aof
+   ```
+   
+   These are the backup files to be restored.
+
+4. Copy the above backup files to the pod to restore them.
+
+   ```
+   kubectl cp ./backups/dump.rdb odim/<pod-name>:/redis-data/dump.rdb-1
```
Comment (ashraf-vazeer) [inline README.md:2388]: Change ./backups/dump.rdb  as follows

<Copied directory>/backups/dump.rdb

File: README.md [hunk 4]
```diff
@@ -2326,11 +2327,166 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
...
+
+   ```
+   kubectl cp ./backups/dump.rdb odim/redis-ha-inmemory-primary-0:/redis-data/dump.rdb-1
+   ```
+
+   ```
+   kubectl cp ./backups/appendonly.aof  odim/redis-ha-inmemory-primary-0/redis-data/appendonly.aof.old
+   ```
+
+5. Log in to redis CLI and disable append-only configuration to restore the data back.
```
Comment (ashraf-vazeer) [inline README.md:2413]: Find out from Shahid, from where this redis-cli has to be executed?

File: README.md [hunk 5]
```diff
@@ -2326,11 +2327,166 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
...
+    ```
+    kubectl exec -it redis-ha-ondisk-primary-0 -nodim bash
+    ```
+    ```
+    kubectl exec -it redis-ha-inmemory-primary-0 -nodim bash
+    ```
+    ```
+    redis-cli --tls --cert /etc/odimra_certs/odimra_server.crt --key /etc/odimra_certs/odimra_server.key --cacert /etc/odimra_certs/rootCA.crt -a <password>
+    ```
+
```
Comment (ashraf-vazeer) [inline README.md:2484]: same as above comment. from where redis-cli has to be executed and what is the command to get into the pod

**Review by ashraf-vazeer — CHANGES_REQUESTED** [review]
please fix the comments and re-submit

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md:2335]: fixed

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md:2357]: fixed

File: README.md (hunk 3 above)
Comment (sunithasomanna) [reply README.md:2388]: fixed

File: README.md (hunk 4 above)
Comment (sunithasomanna) [reply README.md:2413]: fixed

File: README.md (hunk 5 above)
Comment (sunithasomanna) [reply README.md:2484]: fixed

**Comment by sunithasomanna** [comment]
Raised a new PR https://github.com/ODIM-Project/ODIM/pull/1069. Closing this one.

**Pull request closed without merging**